lambdas/process-cri-callback/src/main/java/uk/gov/di/ipv/core/processcricallback/service/CriCheckingService.java

```diff
@@ -233,10 +229,15 @@ public JourneyResponse checkVcResponse(
                             clientOAuthSessionItem.getGovukSigninJourneyId(),
                             callbackRequest.getIpAddress());
 
-            if (ciMitUtilityService.isBreachingCiThreshold(cis)) {
-                return ciMitUtilityService
-                        .getCiMitigationJourneyResponse(cis)
-                        .orElse(JOURNEY_FAIL_WITH_CI);
+            // Check CI levels against the lowest confidence identity requested so we don't send the
```

Comment:
```suggestion
            // Check CI levels for the lowest confidence identity requested so we don't send the
```
Might clarify the comment slightly

Reply:
(I think it's not so much about the mitigation journey specifically, as much as blocking the user with a CI fail)